demos/multi_channel/common/CMakeLists.txt

```diff
@@ -94,22 +91,20 @@ if(MULTICHANNEL_DEMO_USE_NATIVE_CAM)
     message(STATUS "    \"${_LIBVA_INCLUDE_DIR}\"")
 
     target_include_directories(${TARGET_NAME} PRIVATE ${_LIBVA_INCLUDE_DIR})
-    target_link_libraries(${TARGET_NAME}
+    target_link_libraries(${TARGET_NAME} PRIVATE
         X11
         ${_LIBVA_LIB}
         ${_LIBVA_X11_LIB}
         ${_LIBVA_DRM_LIB}
         )
-    target_compile_definitions(${TARGET_NAME} PRIVATE
+    target_compile_definitions(${TARGET_NAME} PUBLIC
         USE_LIBVA=1)
 endif()
 
 target_include_directories(${TARGET_NAME} PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}")
 
+find_package(Threads REQUIRED)
+
 target_link_libraries(${TARGET_NAME}
     PRIVATE ${InferenceEngine_LIBRARIES} gflags ${OpenCV_LIBRARIES}
-    PUBLIC common)
-
-if(UNIX)
-    target_link_libraries(${TARGET_NAME} PRIVATE pthread)
-endif()
+    PUBLIC Threads::Threads common)
```

Comment:
Can you explain to me why `multicam` and `Threads::Threads` are linked publicly now?

Reply:
Because they are used in headers.

Reply:
I see. I've tried compiling with `PRIVATE Threads::Threads` and it passed. That's why I asked. But `main.cpp`s actually have `#include <thread>` even though they don't need it. Probably should be removed later.

Reply:
have you tried to enable TBB for multi channel demo?

Reply:
Yes, passes for Windows with `-DMULTICHANNEL_DEMO_USE_TBB=ON` and `PRIVATE Threads::Threads`.